src/store/modules/position.store.js

```diff
@@ -264,6 +259,37 @@ const actions = {
             )
         }
         if (matchingProjection) {
+            const oldProjection = state.projection
```

Comment:
Should we not check if the projection isalready correct before doing all this work ?

```
if (matchingProjection && matchingProjection.epsg != oldProjection.epsg) {
...
}
```

Reply:
might be a good idea, yes